bitrise.yml

```
@@ -296,6 +297,9 @@ workflows:
 
 app:
   envs:
+  - opts:
+      is_expand: false
+    UITEST_SIMULATOR_DEVICE: iPhone 13 Pro
```

I think, main device is iPhone 13. 
```suggestion
    UITEST_SIMULATOR_DEVICE: iPhone 13
```

---

I am not sure what "main device" refers to and what impact this proposed change has?

I chose the Pro version on the naive assumption that "was the simulator going to provide improve improved performance (as real device would claim), our simulator may theoretically run the tests faster". I don't think this is a valid assumption, but either way, no harm to use the pro version...

What is your motivation for not using the Pro version?

---

I think, we need to agree on with device should run UI test

---

Agreement starts with discussion. From my side I have proposed "iPhone 13 Pro" in the PR and as you have raised the subject I have asked you to have your say.

Please let me know when you'd like to touch this complex subject.